modules/test/playwright/pages/object-web/object-layout/ObjectLayoutsPage.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {FrameLocator, Locator, Page} from '@playwright/test';
+
+import {ViewObjectDefinitionsPage} from '../ViewObjectDefinitionsPage';
+
+export class ObjectLayoutsPage {
+	readonly addBlock: Locator;
+	readonly addField: Locator;
+	readonly addObjectLayoutButton: Locator;
+	readonly addTab: Locator;
+	readonly iframeLocator: FrameLocator;
+	readonly layoutNameBox: Locator;
+	readonly layoutTab: Locator;
+	readonly layoutsTabItem: Locator;
+	readonly page: Page;
+	readonly saveBlockButton: Locator;
+	readonly saveLayoutButton: Locator;
+	readonly saveTabButton: Locator;
+	readonly tabAndBlockNameBox: Locator;
+	readonly viewObjectDefinitionsPage: ViewObjectDefinitionsPage;
+
+	constructor(page: Page) {
+		this.iframeLocator = page.frameLocator('iframe');
+		this.addBlock = this.iframeLocator.getByRole('button', {
+			name: 'Add Block',
+		});
+		this.addField = this.iframeLocator.getByRole('button', {
+			name: 'Add Field',
+		});
+		this.addObjectLayoutButton = page.getByLabel('Add Object Layout');
+		this.addTab = this.iframeLocator.getByRole('button', {name: 'Add Tab'});
+		this.layoutNameBox = page.getByLabel('Name');
+		this.layoutTab = this.iframeLocator.getByRole('tab', {name: 'Layout'});
+		this.layoutsTabItem = page.getByRole('link', {name: 'Layouts'});
+		this.page = page;
+		this.saveBlockButton = this.iframeLocator
+			.getByLabel('Add Block')
+			.getByRole('button', {name: 'Save'});
+		this.saveLayoutButton = page.getByRole('button', {name: 'Save'});
+		this.saveTabButton = this.iframeLocator
+			.getByLabel('Add Tab')
+			.getByRole('button', {name: 'Save'});
+		this.tabAndBlockNameBox = this.iframeLocator.getByLabel('Label');
+		this.viewObjectDefinitionsPage = new ViewObjectDefinitionsPage(page);
+	}
+
+	async createObjectLayout(objectLayoutName: string) {
+		await this.addObjectLayoutButton.click();
+		await this.layoutNameBox.click();
+		await this.layoutNameBox.fill(objectLayoutName);
+		await this.saveLayoutButton.click();
+	}
+
+	async goto(objectDefinitionLabel: string) {
+		await this.viewObjectDefinitionsPage.goto();
+
+		await this.viewObjectDefinitionsPage.clickEditObjectDefinitionFDSLink(
+			objectDefinitionLabel
+		);
+
+		await this.layoutsTabItem.click();
+	}
+
+	async openObjectLayout(objectLayoutName: string) {
+		await this.page.getByRole('link', {name: objectLayoutName}).click();
+		await this.layoutTab.click();
+		await this.addTab.click();
+		await this.tabAndBlockNameBox.click();
+		await this.tabAndBlockNameBox.fill(objectLayoutName);
```

Comment:
Hey @vngoliveira, I can see some click's before the fill function. Although these click was not needed, since the fill method already focus the element.

Check the fill funciton playwright details:

> Details
> 
> This method waits for [actionability](vscode-file://vscode-app/usr/share/code/resources/app/out/vs/code/electron-sandbox/workbench/workbench.html) checks, focuses the element, fills it and triggers an input event after filling. Note that you can pass an empty string to clear the input field.
> 
> If the target element is not an <input>, <textarea> or [contenteditable] element, this method throws an error. However, if the element is inside the <label> element that has an associated [control](vscode-file://vscode-app/usr/share/code/resources/app/out/vs/code/electron-sandbox/workbench/workbench.html), the control will be filled instead.